src/api.v2/controllers/sampleFileController.js

```diff
@@ -0,0 +1,55 @@
+const sqlClient = require('../../sql/sqlClient');
+
+const Sample = require('../model/Sample');
+const SampleFile = require('../model/SampleFile');
+
+const { getSampleFileUploadUrl } = require('../helpers/s3/signedUrl');
+const { OK } = require('../../utils/responses');
+const getLogger = require('../../utils/getLogger');
+
+const logger = getLogger('[SampleFileController] - ');
+
+const createFile = async (req, res) => {
+  const {
+    params: { experimentId, sampleId, sampleFileType },
+    body: { sampleFileId, size, metadata = {} },
+  } = req;
+  logger.log(`Creating sample file for experiment ${experimentId}, sample ${sampleId}, sampleFileType ${sampleFileType}`);
+
+  const newSampleFile = {
+    id: sampleFileId,
+    sample_file_type: sampleFileType,
+    size,
+    s3_path: sampleFileId,
+    upload_status: 'uploading',
+  };
+
+  let signedUrl;
+
+  await sqlClient.get().transaction(async (trx) => {
+    await new SampleFile(trx).create(newSampleFile);
+    await new Sample(trx).setNewFile(sampleId, sampleFileId, sampleFileType);
+
+    signedUrl = getSampleFileUploadUrl(sampleFileId, metadata);
+  });
+
+  logger.log(`Finished creating sample file for experiment ${experimentId}, sample ${sampleId}, sampleFileType ${sampleFileType}`);
```

Comment:
Maybe `Created file {sampleFileType} for sample {sample} in experiment {experimentId}`

Reply:
I added finished whenever we finish any controller function because I think it will be easier to spot in the logs if we are consistently writing "Finished" whenever something finishes